build_skymap.sh

```diff
@@ -1,14 +1,12 @@
-#!/bin/bash
+#!/bin/sh -e
```

Comment:
Could you add more explanation of why some of the script lines changed that aren't related to the build flavors?